README.md

```diff
@@ -9,7 +9,17 @@
 
 ## Releases
 
-Refer to [Github releases](https://github.com/UN-OCHA/common_design/releases) for latest updates. We use [npm-version](https://docs.npmjs.com/cli/version) and [semantic versioning](https://semver.org/)
+We use [npm-version](https://docs.npmjs.com/cli/version) and [semantic versioning](https://semver.org/)
+
+We aim to facilitate an annual major release. The next release date is June 2022.
+
+We continue to regularly release patches or minor releases to include bug fixes, dependency updates, and component 
+improvements. Refer to [Github releases](https://github.com/UN-OCHA/common_design/releases) for the latest tagged 
+releases. 
+
+There may be several breaking changes during the year which result in increments to the major release number. Teams 
```

Comment:
@left23 how would you prefer to manage these docs?

1. Should we start a CHANGELOG that actually lists tickets being fixed, improvements added, breaking changed documented?
2. Should we carefully document each GitHub Release with the notes, and then compile them annually?
3. Another approach I haven't considered?

My thinking is that a CHANGELOG file is a bit more work, but it's constantly updated and considered ready-to-ship on our side, meaning nothing will be forgotten over time. Compiling them from GH Releases once per year might result in omissions or just plain forgetting to write something.

Reply:
An example from recent happenings:

[CD 4.0.0 release notes mention removing Gulp](https://github.com/UN-OCHA/common_design/releases/tag/v4.0.0), but doesn't document remediation. Having discussed this on the CD call this week and subsequently [re-reading my own notes in a flowdock thread](https://www.flowdock.com/app/unocha/common-design/threads/E8tDGDMCCzU3JKhLv62Wq9MnVxv), I have an overview of what needs to change when upgrading to 4.0.0.

Will I remember the fine details in May 2022 when we prepare to communicate to others? I don't trust me that much :)

Reply:
Good questions @rupl
I think some valuable experience has been gained through the process so far, so some of the difficulties this time around I don't expect will repeat.

For clarity, the last few months we released two major releases - 3 and 4. This means teams would be moving from 2 to 4.
There's more detail about removing Gulp in the v3.1.0 https://github.com/UN-OCHA/common_design/releases?after=v3.1.0 but I agree, it's unclear and hard to find the relevant info on what has to change in the wider site repo (The release notes so far have dealt mainly with what's happening in the theme directory)

The reality is all sites using v2 have been updated to v4 by now (bar FTS, in progress. I've provided additional documentation and a screenshare session) so we're talking about the future from this point on. We can assume the release in 2022 with be v5 or upwards.

I had envisioned using the Github Releases UI but maybe with a more consistent format. I think you'd suggested patches could be tagged releases only, and minor and major releases could also have proper Github Releases where we go into detail. I often link to specific PRs in the Release description. 

I don't have a preference but I am all for having this not be ambiguous. Do we know of other projects DS manages that involve this level of versioning and cater for multiple implementations being dependent?

Reply:
@left23 Even from an ops perspective we don't have explicit versioning for our projects. It would be based on the "upstream" project (e.g., the version of PHP or NewRelic agent that's installed, etc). I'm most familiar with Drupal.org projects where the release notes are inconsistent at best.

I've started poking around a bit on https://design-system.service.gov.uk/. It's matured A LOT since I first looked. I'm trying to figure out how they version and release things. Their prototyping kit is on [version 7](https://design-system.service.gov.uk/get-started/prototyping/) but I don't see any release notes or differences from version 6. The release notes for their front end system [are handled in GitHub exclusively](https://github.com/alphagov/govuk-frontend/releases/tag/v3.12.0) (as best I can see...I used the [link from the install notes to come to this conclusion](https://frontend.design-system.service.gov.uk/install-using-precompiled-files/#copy-the-files)). Their "just install a node package" seems like the WFP setup and I feel it glosses over how to implement any required changes in markup.

@berliner and @s0 may also have some ideas on how to collate this information. They are currently our biggest "consumers" of the releases so it would be good to get ideas on how to make this information useful. Quick reminder for both: OCHA is in the process of implementing a formal brand standard policy. As part of this change the Strategic Communications branch (product owner for the common design, PPT templates, etc) will have an annual "refresh" window where teams should look to download and apply the latest version of their templates. This also gives a predictable window for teams using these templates to set aside time to refresh their designs. (All good so far.) HOWEVER, in the meantime, Digital Services (as technical implementors of the Common Design) will be tinkering and making improvements. We use semantic versioning and expect to have multiple releases throughout the year. On the "annual refresh" window what is the best way we can communicate what has changed over the year?

PS @left23 should I also copy any of this back to the [Jira issue](https://humanitarian.atlassian.net/browse/CD-301)?

Reply:
If I could wish for something, I'd opt for a CHANGELOG.txt as part of the repo as well as notes in the github releases and update instructions from the last major version. This would help us a lot to get the updates done quickly.

Reply:
Agreed with what @berliner has said :)